src/httpd_jsonapi.c

```diff
@@ -706,6 +712,10 @@ jsonapi_reply_config(struct httpd_request *hreq)
     }
   json_object_object_add(jreply, "directories", directories);
 
+  // Config for creating/modifying stored playlists
+  json_object_object_add(jreply, "allow_modifying_stored_playlists", json_object_new_boolean(cfg_getbool(lib, "allow_modifying_stored_playlists")));
+  json_object_object_add(jreply, "default_playlist_directory", json_object_new_string(cfg_getstr(lib, "default_playlist_directory")));
```

Comment:
we have a `SEGV` here - the `default_playlist_directory` may be unset which means the value coming back from `cfg_getstr(...)` will be `NULL`